debian/control

```diff
@@ -233,7 +233,9 @@ Provides: miral-examples (=1.5)
 Depends: ${misc:Depends},
          ${shlibs:Depends},
 Recommends: dmz-cursor-theme,
-            fonts-liberation
+            fonts-liberation,
+            qtwayland5,
+            xwayland
```

Comment:
```suggestion
            xwayland,
```
(This way the next diff won't have a `-`)
Otherwise, good.

Reply:
Ack